components/gc-features/reports/a11y-1-fr.html

```diff
@@ -0,0 +1,16 @@
+---
+{
+	"title": "Pré-assessment de l'accessibilité #1 - Promotions contextuelles",
+	"language": "fr",
+	"description": "Partielle. Pré-évaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
+	"tag": "gc-features",
+	"parentdir": "gc-features",
+	"parentPage": "Promotions contextuelles",
+	"parentPageURL": "gc-features",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-04",
+	"layout": "assessment_wrote_en-fr",
+	"reportURL": "a11y-1.json"
```

Comment:
The name need to be adjusted.

```suggestion
	"reportURL": "pre-a11y-1.json"
```

Reply:
Fixed